File: apps/dotnet/Simex/Simex/Models/City.cs
using System;
using System.Collections.Generic;

namespace Simex.Models;

public partial class City
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public int CountryId { get; set; }

    public int? RegionId { get; set; }

    public decimal? Latitude { get; set; }

    public decimal? Longitude { get; set; }

    public int? Altitude { get; set; }

    public virtual ICollection<Airport> Airports { get; set; } = new List<Airport>();

    public virtual ICollection<Carrier> Carriers { get; set; } = new List<Carrier>();

    public virtual ICollection<Company> Companies { get; set; } = new List<Company>();

    public virtual Country Country { get; set; } = null!;

    public virtual ICollection<Port> Ports { get; set; } = new List<Port>();

    public virtual Region? Region { get; set; }

    public virtual ICollection<ShippingLine> ShippingLines { get; set; } = new List<ShippingLine>();
}

File: apps/web/backend/app/Models/City.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class City extends Model
{
    protected $table = 'CITIES';

    protected $primaryKey = 'ID';

    public $timestamps = false;

    protected $fillable = [
        'NAME',
        'COUNTRY_ID',
        'REGION_ID',
        'LATITUDE',
        'LONGITUDE',
        'ALTITUDE',
    ];

    public function country(): BelongsTo
    {
        return $this->belongsTo(Country::class, 'COUNTRY_ID', 'ID');
    }

    public function region(): BelongsTo
    {
        return $this->belongsTo(Region::class, 'REGION_ID', 'ID');
    }
}
